main.go

```diff
@@ -237,10 +265,8 @@ func initGame(database *sql.DB, cfg config.Config) (err error) {
 	}
 	log.Println("Flag timeout:", scoreboard.FlagTimeout)
 
-	scoreboardRecalcD := cfg.Scoreboard.RecalcTimeout.Duration
-	if scoreboardRecalcD != 0 {
-		scoreboard.ScoreboardRecalcTimeout = scoreboardRecalcD
-	}
+	checkScoreboardRecalcTimeout(cfg)
```

Comment:
This is less clear than in the original version.

The goal is to improve readability, no just for achive 100% score at goreportcard :)

Reply:
Okay. I thought that the main problem is cyclomatic complexity, but I was wrong. I return scoreboardRecal if and made new function for checking tasks prices. IMHO, it has improved readability.